src/containers/trends/TrendsContainer.js

```diff
@@ -342,17 +341,24 @@ export class TrendsContainer extends React.Component {
   }
 }
 
-export function mapStateToProps(state) {
+export function mapStateToProps(state, ownProps) {
+  const userId = _.get(ownProps, 'currentPatientInViewId');
   return {
-    viz: state.viz,
+    trendsState: _.get(state, ['viz', 'trends', userId], {}),
   };
 }
 
-export function mapDispatchToProps(dispatch) {
+export function mapDispatchToProps(dispatch, ownProps) {
   return bindActionCreators({
-    focusTrendsCbgSlice: actions.focusTrendsCbgSlice,
-    markTrendsViewed: actions.markTrendsViewed,
-    unfocusTrendsCbgSlice: actions.unfocusTrendsCbgSlice,
+    focusTrendsCbgSlice: actions.focusTrendsCbgSlice.bind(
+      null, ownProps.currentPatientInViewId
```

Comment:
I would prefer the `_.partial` over the `.bind(null, ...)` as they are actually doing slightly different things and I was sort of left wondering why we needed to drop the execution context from this set of binds.